src/domain/Processors/VehicleProcessor.ts

```diff
@@ -429,7 +427,7 @@ export abstract class VehicleProcessor<T extends Vehicle> {
                           );
     errors = errors.concat(this.validateTechRecordFields(newVehicle.techRecord[0], isCreate));
     if (errors && errors.length) {
-      console.error(errors);
+      // console.error(errors);
```

Comment:
:( think logging errors is good. if its really not needed remove the line.

Reply:
I did it to temporarily remove the logs as I was debugging other logs :). Forgot to uncomment. Thanks for highlighting.